Plugins/Gerrit/GerritPlugin.cs

```diff
@@ -98,7 +98,7 @@ private bool HaveValidCommitMsgHook([NotNull] IGitModule gitModule, bool force)
         {
             if (gitModule == null)
             {
-                throw new ArgumentNullException("gitDirectory");
+                throw new ArgumentNullException(nameof(gitModule));
```

Comment:
Previous code likely an artefact of refactoring. New code shouldn't have that problem in future.